gtk/src/main.c

```diff
@@ -246,12 +246,12 @@ bind_audio_tree_model(signal_user_data_t *ud)
     gtk_tree_view_column_set_max_width(column, 400);
 
     column = gtk_tree_view_column_new_with_attributes(
-                                    _(""), edit_cell, "icon-name", 3, NULL);
+                                    _(" "), edit_cell, "icon-name", 3, NULL);
```

Comment:
What about doing
```suggestion
                                    "", edit_cell, "icon-name", 3, NULL);
```
? An empty string doesn't need to be translated🙂

Reply:
Actually, I think using a space char here is appropriate.  If someone were to ever add a label here, the macro would still be in place and they would not inadvertently create another missing localization string issue.